templates/bake/element/form.twig

```diff
@@ -188,7 +185,7 @@
 <template>
     <Layout>
         {%- set vueVar = "{{ props.flash['message'] }}" %}{{- "\n" }}
-        <div onclick="this.classList.add('d-none')" :class="[(props.flash['element'] === undefined) ? 'd-none' : '','alert', props.flash['element']]">
+        <div id="flash" onclick="this.classList.add('d-none')" :class="[(props.flash['element'] === undefined) ? 'd-none' : '','alert', props.flash['element']]">
```

Comment:
id would need to be unique in the page, I would use a longer id

Reply:
removed, not used